res/values-de/strings.xml

```diff
@@ -0,0 +1,386 @@
+    <string name="event_pull_request_comment">[b]%1$s[/b] kommentierte den Pull-Request [b]%2$d[/b] für [b]%3$s[/b]</string>
+    <string name="event_team_user_add">[b]%1$s[/b] fügte [b]%2$s[/b] zum Team [b]%3$s[/b] hinzu</string>
+    <string name="event_team_repo_add">[b]%1$s[/b] fügte die Repository [b]%2$s[/b] zum Team [b]%3$s[/b] hinzu</string>
+    <plurals name="page">
+        <item quantity="one">eine Seite</item>
+        <item quantity="other">%d Seiten</item>
+    </plurals>
+
+    <string name="event_pull_request_desc">%1$s\n%2$d Beiträge mit %3$d neuen Zeilen und %4$d gelöschten Zeilen</string>
+    <string name="event_follow_desc">%1$s hat %2$d Repositories und %3$d Follower</string>
+    <string name="event_fork_desc">Die Repository %1$s wurde angelegt</string>
+    <string name="event_create_repo_desc">Die neue Repository befindet sich bei %1$s</string>
+    <string name="event_create_branch_desc">Die neue Branch %1$s befindet sich bei %2$s/tree/%3$s</string>
+    <string name="event_gollum_desc">Siehe %1$s</string>
+    <string name="event_push_desc">%1$d zusätzliche Beiträge\u2026</string>
+
+    <string name="user_username">Benutzername</string>
+    <string name="user_password">Passwort</string>
+    <string name="user_following">Folgt</string>
+    <string name="user_followers">Follower</string><!-- For type organization -->
+    <string name="user_members">Mitglieder</string><!-- For type organization -->
+    <string name="user_created_at">Mitglied seit %1$s</string>
+    <string name="user_pub_repos">Repositories</string>
+    <string name="user_extra_data">%1$d Follower\u00a0\u00a0\u00a0%2$d Repositories</string>
+    <string name="user_news_feed">Neuigkeiten</string>
+    <string name="user_your_actions">Aktivitäten</string>
+    <string name="user_public_activity">Öffentliche Aktivitäten</string>
+    <string name="user_organizations">Organisationen</string>
+    <string name="user_follow_action">Folgen</string>
+    <string name="user_unfollow_action">Entfolgen</string>
+    <string name="user_no_repos">Es wurden keine Repositories gefunden</string>
+    <string name="user_no_data">Für diesen Benutzer konnten keine Daten geladen werden.</string>
+
+    <!-- Repository -->
+    <string name="repository">Repository</string>
+    <string name="repo_forks">Forks</string>
+    <string name="repo_language">Sprache %1$s</string>
+    <string name="repo_watchers">Follower</string>
+    <string name="repo_stargazers">Favorisierende</string>
+    <string name="repo_watch_action">Folgen</string>
+    <string name="repo_star_action">Favorisieren</string>
+    <string name="repo_select_ref">Branch auswählen</string>
+    <string name="repo_select_ref_dialog_title">Branch oder Tag auswählen</string>
+    <string name="repo_branches">Branches</string>
+    <string name="repo_tags">Tags</string>
+    <string name="repo_unwatch_action">Entfolgen</string>
```

Comment:
In der ActionBar wird der String aber nie angezeigt (da 'withText' nicht gesetzt ist), sondern nur für den Tooltip beim langen Draufdrücken und in der Menüanzeige (bei Geräten mit Menütaste) verwendet. Daher ist korrektes Deutsch IMHO wichtiger ;)